blocks/redirects-formatter/redirects-formatter.css

```diff
@@ -0,0 +1,88 @@
+
+:root {
+  --notch-size: 12px;
+}
+
+h1 {
```

Comment:
We usually specify selectors with the block class, but I guess you don't foresee this block being used on a page with other blocks?